src/utils/NetworkUtils.ts

```diff
@@ -1,39 +1,22 @@
-import { PublicNetworks } from "@uma/common/dist/PublicNetworks";
-import { CHAIN_IDs, PRODUCTION_CHAIN_IDS, TESTNET_CHAIN_IDS } from "../constants";
-
-/**
- * A list of networks that provide more resolution about a chainid -> network name
- */
-const networkIdMap: Record<number, string> = {
-  666: "Hardhat1",
-  1337: "Hardhat2",
-  421613: "ArbitrumGoerli",
-  421614: "ArbitrumSepolia",
-  324: "ZkSync",
-  280: "ZkSync-Goerli",
-  300: "ZKSync-Sepolia",
-  8453: "Base",
-  84531: "BaseGoerli",
-  84532: "BaseSepolia",
-  59144: "Linea",
-  59140: "LineaGoerli",
-  11155111: "EthSepolia",
-  11155420: "OptimismSepolia",
-};
+import { CHAIN_IDs, PRODUCTION_CHAIN_IDS, PUBLIC_NETWORKS, TESTNET_CHAIN_IDS } from "../constants";
 
 /**
  * Resolves a network name from a network id.
  * @param networkId The network id to resolve the name for
  * @returns The network name for the network id. If the network id is not found, returns "unknown"
  */
 export function getNetworkName(networkId: number | string): string {
+  networkId = Number(networkId);
   try {
-    const networkName = PublicNetworks[Number(networkId)].name;
+    const networkName = PUBLIC_NETWORKS[networkId].name;
     return networkName.charAt(0).toUpperCase() + networkName.slice(1);
```

Comment:
@dohaki @james-a-morris Are we dependent on this behaviour anywhere in the FE? I'd rather zap it and just inherit the naming specified in constants-v2 but just want to make sure that it doesn't break anything.

Reply:
No, AFAICT we don't use that function in the FE